amundsen_application/static/js/config/config-types.ts

```diff
@@ -6,23 +6,27 @@
 
 export interface AppConfig {
   browse: BrowseConfig;
+  editableText: EditableTextConfig;
   google: GoogleAnalyticsConfig;
+  indexUsers: IndexUsersConfig;
   logoPath: string | null;
+  mailClientFeatures: MailClientFeaturesConfig;
   navLinks: Array<LinkConfig>;
   resourceConfig: ResourceConfig;
   tableLineage: TableLineageConfig;
   tableProfile: TableProfileConfig;
-  indexUsers: indexUsersConfig;
 }
 
 export interface AppConfigCustom {
   browse?: BrowseConfig;
+  editableText?: EditableTextConfig;
   google?: GoogleAnalyticsConfig
+  indexUsers?: IndexUsersConfig;
   logoPath?: string;
+  mailClientFeatures: MailClientFeaturesConfig;
```

Comment:
This should be optional, otherwise an error will be thrown if your private config is not configured with this field. Eventually I think we could do something with `Partial<>` and optional `?` and smart object merging to make allow default values and custom values to work more seamlessly.

Reply:
Yeah this is an oversight on master as well that would be undesirable in the notifications release. Let me change it there and pull the change in.